zulu/datetime.py

```diff
@@ -567,6 +568,114 @@ def span(self, frame, count=1):
         """
         return (self.start_of(frame), self.end_of(frame, count))
 
+    @classmethod
+    def validate_times(cls, start, end):
```

Comment:
These validation methods would be better as regular functions. They don't really depend on the class so they are fine being standalone.